test/SILGen/copy_expr.swift

```diff
@@ -0,0 +1,85 @@
+// RUN: %target-swift-emit-silgen %s | %FileCheck %s
+
+final class Klass {
+  var k: Klass? = nil
+}
+struct ContainKlass {
+  var k = Klass()
+}
+
+// CHECK-LABEL: sil hidden [ossa] @$s9copy_expr22testCopyLoadableRValueyyF : $@convention(thin) () -> () {
+// CHECK: [[X:%.*]] = apply {{%.*}}({{%.*}}) : $@convention(method) (@thin ContainKlass.Type) -> @owned ContainKlass
+// CHECK: [[BORROW:%.*]] = begin_borrow [lexical] [[X]]
+// CHECK: [[COPY_BORROW:%.*]] = copy_value [[BORROW]]
+// CHECK: explicit_copy_value [[COPY_BORROW]]
+// CHECK: } // end sil function '$s9copy_expr22testCopyLoadableRValueyyF'
+func testCopyLoadableRValue() {
```

Comment:
I think it's worth covering cases where the copy's result is passed to `consuming`, `borrowing`, and `inout` parameters to ensure the explicit copy is there. For example, `(copy x).prop` where `prop` has a `__consuming get`, or a regular `get`.

Reply:
@kavon I added a bunch below.